bundles/org.eclipse.passage.loc.licenses.core/src/org/eclipse/passage/loc/internal/licenses/core/IssueFloatingLicense.java

```diff
@@ -120,6 +124,7 @@ private ServiceInvocationResult<List<Path>> persist(EObject target, LicensedProd
 		Path lic;
 		try {
 			lic = new PersistedDecoded(folder, target).write(decrypted);
+			events.sendEvent(new OperatorLicenseEvents().decodedIssued(lic.toString()));
```

Comment:
Btw. what do you think about passing the `Path` object directly, here and below?